plugins/karate/Makefile.mk

```diff
@@ -1,8 +1,10 @@
 # LIBRARIES
 ##################################################
 if USE_KARATE
+BUILT_SOURCES += plugins/karate/PluginDescription.h
```

Comment:
I think that this is not possible. Especially if we want to respect the `USE_KARATE` etc. variables.